pkg/controller/capacity/builder/report_test.go

```diff
@@ -0,0 +1,326 @@
+package builder
+
+import (
+	"github.com/pmezard/go-difflib/difflib"
```

Comment:
A minor remark: this package is no longer maintained, we might consider forking it or using an alternative.

Reply:
We already vendor it in the project. If required, we could fork it into bookingcom organization and change the import path.

Do you think this is relevant for this PR?